packages/app/src/components/SearchBar/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
+import { NominationsDataContext } from '../../utils/context/NominationsContext';
```

Comment:
What's the reasoning for using React's context API? From [the docs](https://reactjs.org/docs/context.html#before-you-use-context), contexts are most useful when sharing data across many components at different nesting levels.

Given we're only using the list of nominations in this component, I think there may be a simpler way to implement this functionality.